CHANGELOG.md

```diff
@@ -1,71 +1,120 @@
- [#500](https://github.com/mattermost/desktop/issues/500)
- - Added the dialog to reopen the application when it quits unexpectedly.
+ - Added a dialog to allow the user to reopen the desktop app if it quits unexpectedly.
  [#626](https://github.com/mattermost/desktop/pull/626)
- - Show animation icon when loading a page.
+ - Mattermost animation icon is now displayed when loading a page, instead of a blank screen.
  [#490](https://github.com/mattermost/desktop/issues/490)
- - Show the dialog to confirm requested permissions.
+ - Added a dialog to request permissions to show desktop notifications or to use microphone and video for [video calls](https://docs.mattermost.com/deployment/webrtc.html) from untrusted origins.
  [#609](https://github.com/mattermost/desktop/pull/609)
+ - The "Saved" indicator now appears for both Server Management and App Options on the Settings page.
+ [#500](https://github.com/mattermost/desktop/issues/500)
+ - Close button on the Settings page now has a hover effect.
+[#439](https://github.com/mattermost/desktop/issues/439)
+ - Added new admin configuration settings ([#586](https://github.com/mattermost/desktop/pull/586) & [#633](https://github.com/mattermost/desktop/pull/633)) for
+   - Disabling server management where the user cannot add or edit the server URL.
+   [#600](https://github.com/mattermost/desktop/pull/600)
+   - Setting one or more pre-configured server URLs for the end user.
+   [#594](https://github.com/mattermost/desktop/pull/594)
+   - Customizing the link in **Help > Learn More..**.
+   [#593](https://github.com/mattermost/desktop/pull/593)
 
 #### Windows
- - Added the feature to open the application via `mattermost://` link.
+ - Added support for protocol deep linking where the desktop app opens via `mattermost://` link if app is already installed.
  [#616](https://github.com/mattermost/desktop/pull/616)
+ - Added the ability to more easily whitelabel the Mattermost taskbar icon on custom builds.
+ [#592](https://github.com/mattermost/desktop/pull/592)
 
 #### Mac
- - Added `Ctrl+Tab` and `Ctrl+Shift+Tab` shortcuts to switch tabs
- [#512](https://github.com/mattermost/desktop/issues/512)
- - Added the feature to open the application via `mattermost://` link.
+ - Added support for protocol deep linking where the desktop app opens via `mattermost://` link if app is already installed.
  [#616](https://github.com/mattermost/desktop/pull/616)
+ - Added `Ctrl+Tab` and `Ctrl+Shift+Tab` shortcuts to switch between server tabs,
+ [#512](https://github.com/mattermost/desktop/issues/512)
  - Added the option to bounce the Dock icon when receiving new messages.
  [#514](https://github.com/mattermost/desktop/issues/514)
 
+### Architectural Changes
+ - Major version upgrade of Electron from v1.6.11 to v1.7.9. Electron is the underlying technology used to build the Desktop apps.
+ [#602](https://github.com/mattermost/desktop/pull/602)
+ - The app now employs Mattermost APIv4 endpoints.
```

Comment:
I think that API is not used yet. It will be used for #608 to detect whether the user is logged in.